docker/docker-compose.yml

```diff
@@ -54,8 +68,12 @@ services:
   embedder:
     image: embedder
     depends_on:
-      - postgres
-      - ollama
+      postgres:
+        condition: service_started
```

Comment:
As far as i know, `postgres` has a built-in `service_healthy` support.

Reply:
I am not sure when I did condition: service_healthy, I was getting an error that said: 'container docker-postgres-1 has no healthcheck configured'